internal/utils/utils.go

```diff
@@ -0,0 +1,232 @@
+package utils
+
+import (
+	_ "embed"
+
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+	"text/template"
+
+	"github.com/paketo-community/ubi-nodejs-extension/structs"
+
+	"github.com/BurntSushi/toml"
+)
+
+var DEFAULT_USER_ID = 1002
+var DEFAULT_GROUP_ID = 1000
```

Comment:
I wonder if these could be in a common place as we define them in multiple files?